lib/elixir/src/elixir_module.erl

```diff
@@ -351,7 +351,8 @@ attributes(DataSet, DataBag, PersistedAttributes) ->
 lookup_attribute(DataSet, DataBag, Key) when is_atom(Key) ->
   case ets:lookup(DataSet, Key) of
     [{Key, _, accumulate}] -> bag_lookup_element(DataBag, {accumulate, Key}, 2);
-    [{Key, Value, _}] -> [Value];
+    [{Key, nil, unset}] -> [];
+    [{_, Value, _}] -> [Value];
```

Comment:
I tried matching on `is_integer(Line)` here, but it seems something with protocols adds a value, where the third element is `nil`.

Reply:
```
== Compilation error in file lib/calendar/date_range.ex ==
** (CaseClauseError) no case clause matching: [{:protocol_impl, [protocol: Enumerable, for: Date.Range], nil}]
    lib/calendar/date_range.ex:26: (module)
```

Reply:
The value is nil every time a module attribute is read. Check `__get_attribute__` impl.